hazelcast-sql/src/main/java/com/hazelcast/jet/sql/impl/connector/map/IMapSqlConnector.java

```diff
@@ -188,12 +160,34 @@ public VertexWithInputConfig nestedLoopReader(
         return IMapJoiner.join(dag, name, toString(table), joinInfo, rightRowProjectorSupplier);
     }
 
+    @Nonnull
+    @Override
+    public Vertex fullScanReader(
```

Comment:
Could we keep the interface method order?

Reply:
Can I move it in next PR (index scan) ?